src/maud/sampling.py

```diff
@@ -328,17 +343,35 @@ def get_input_data(
         )
         for measurement_type in ["metabolite", "reaction", "enzyme"]
     )
-
-    conc_init = pd.DataFrame(
-        0.01, index=experiment_codes.values(), columns=mic_codes.values()
-    )
     for _i, row in mic_measurements.iterrows():
         if row["target_id"] in balanced_mic_codes.keys():
             row_ix = experiment_codes[row["experiment_id"]]
             column_ix = mic_codes[row["target_id"]]
             conc_init.loc[row_ix, column_ix] = row["value"]
+    for _i, row in enzyme_measurements.iterrows():
+        row_ix = experiment_codes[row["experiment_id"]]
+        column_ix = enzyme_codes[row["target_id"]]
+        enz_conc_init.loc[row_ix, column_ix] = row["value"]
+    for prior_unb in mi.priors.unbalanced_metabolite_priors:
+        mic_id = mic_codes[prior_unb.mic_id]
+        exp_id = experiment_codes[prior_unb.experiment_id]
+        conc_init.loc[exp_id, mic_id] = prior_unb.location
+    for prior_enz in mi.priors.enzyme_concentration_priors:
+        enz_id = enzyme_codes[prior_enz.enzyme_id]
+        exp_id = experiment_codes[prior_enz.experiment_id]
+        enz_conc_init.loc[exp_id, enz_id] = prior_enz.location
```

Comment:
I think it's possible for there to be some overwriting here, so maybe it's better to put the priors before the measurements? I think if an enzyme or metabolite has a prior and a measurement then the measurement will probably be the better initial value.